client/app/telecom/sms/sms/compose/telecom-sms-sms-compose.html

```diff
@@ -341,7 +341,7 @@ <h2 data-translate="loading"></h2>
                     <div class="form-group">
                         <button
                             type="button"
-                            class="btn btn-sm btn-link no-style p-0 telecom-link"
+                            class="btn btn-sm btn-link p-0 telecom-link"
```

Comment:
I think you have to remove `btn-sm` since we need to have same button sizes everywhere

Reply:
@GenghisJean a `btn btn-link` is just a `oui-button oui-button_link`.
Here `btn-sm` helps to cut down some padding then we can stick with it.